mkdocs.yml

```diff
@@ -5,6 +5,8 @@ repo_name: kubernetes/cloud-provider-aws/
 repo_url: https://github.com/kubernetes/cloud-provider-aws/
 nav:
   - 'Home': README.md
+  - 'Components':
```

Comment:
This is minor, but I think these tabs are listed by the order they have here. We might want to move it to between user guide and development.

Reply:
That makes sense.  We can do in a follow up.